megamek/src/megamek/common/weapons/MissileWeaponHandler.java

```diff
@@ -1150,4 +1150,13 @@ protected boolean canDoDirectBlowDamage() {
     protected boolean isAdvancedAMS() {
         return advancedAMS && (amsEngaged || apdsEngaged);
     }
+    
+    //Check for Thunderbolt. We'll use this for single AMS resolution
+    @Override
+    protected boolean isTbolt() {
+        if (wtype.hasFlag(WeaponType.F_LARGEMISSILE)) {
+            return true;
+        }
+        return false;
+    }
```

Comment:
Looks like some tab characters have been introduced into this file as well.

Reply:
I don't see the problem here. I have spaces when I open this...

Reply:
Looks like you introduced them in an earlier check-in.  Take a look up around lines 933-983.

Reply:
Fixed.

Reply:
Okay, last thing that needs to be done, near as I can tell, is that the conflicts need to be resolved before we can do the actual merge.  Hopefully we can get @arlith checking this out as well.

Reply:
oh, ok.  I thought the conflicts were on your side. Probably a recent commit that I didn't pull in.

Reply:
Hah yeah.  LOTS of recent commits. I show a green light on conflicts now.